test/browser_tests/page_objects/page_careers-application-process.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var _assign = require( 'lodash' ).assign;
```

Comment:
Should we pick one, underscore or lodash, so that's it's consistent throughout the project?